pkg/util/tags.go

```diff
@@ -0,0 +1,174 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package util
```

Comment:
In a separate commit (f8156c394b7aff97026600816d7150a06777e1be) i moved the Tags comparison/update logic to separate package, we can reuse it for `StateMachines` as well. Thoughts?